data/sql/updates/pending_db_world/rev_1633275809062975000.sql

```diff
@@ -0,0 +1,7 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1633275809062975000');
+
+-- baron adds 
+UPDATE `creature_onkill_reputation` SET `RewOnKillRepValue1` = 1 WHERE (`creature_id` = 11197);
+
+-- scholo projections 
+UPDATE `creature_onkill_reputation` SET `RewOnKillRepValue1` = 1 WHERE (`creature_id` = 11263);
```

Comment:
add line at the end of file
```suggestion
UPDATE `creature_onkill_reputation` SET `RewOnKillRepValue1` = 1 WHERE (`creature_id` = 11263);

```